src/utils/netCoreUtils.ts

```diff
@@ -11,11 +11,11 @@ import { ext } from '../extensionVariables';
 import { execAsync } from './execAsync';
 import { getTempFileName } from './osUtils';
 
-export async function getNetCoreProjectInfo(target: 'GetBlazorManifestLocations' | 'GetProjectProperties', project: string): Promise<string[]> {
+export async function getNetCoreProjectInfo(target: 'GetBlazorManifestLocations' | 'GetProjectProperties', project: string, additionalProperties?: string): Promise<string[]> {
     const targetsFile = path.join(ext.context.asAbsolutePath('resources'), 'netCore', `${target}.targets`);
     const outputFile = getTempFileName();
 
-    const command = `dotnet build /r:false /t:${target} /p:CustomAfterMicrosoftCommonTargets="${targetsFile}" /p:CustomAfterMicrosoftCommonCrossTargetingTargets="${targetsFile}" /p:InfoOutputPath="${outputFile}" "${project}"`;
+    const command = `dotnet build /r:false /t:${target} /p:CustomAfterMicrosoftCommonTargets="${targetsFile}" /p:CustomAfterMicrosoftCommonCrossTargetingTargets="${targetsFile}" /p:InfoOutputPath="${outputFile}" ${additionalProperties} "${project}"`;

```

Comment:
We'll need to use ${additionalProperties || ''} to make sure we don't end up with `unknown` in the string output if additionalProperties is undefined.